jina/docker/hubio.py

```diff
@@ -339,7 +340,8 @@ def build(self) -> Dict:
                 image, log = self._client.images.build(path=self.args.path,
                                                        tag=self.tag,
                                                        pull=self.args.pull,
-                                                       dockerfile=self.dockerfile_path_revised,
+                                                       dockerfile=self.dockerfile_path,
+                                                       labels={k: str(v) for k, v in self.manifest.items()},
```

Comment:
is self.manifest not enough?

Reply:
No, everything needs to be serialised into a string